<launch>
  <node pkg="calib_wide_camera" type="bag2img" name="bag2img" output="screen">
    <param name="bag_path" type="string" value="/home/sam/Desktop/lht/in/2023-09-08-16-44-26.bag"/>
    <param name="write_path" type="string" value="/home/sam/Desktop/lht/in/up/"/>
    <param name="topic_name" type="string" value="/usb_cam_up/image_raw/compressed"/>
    <param name="isCompressed" type="bool" value="1"/>
  </node>
</launch>